packages/cashier/src/pages/withdrawal/crypto-withdraw-receipt/crypto-withdraw-receipt.tsx

```diff
@@ -29,14 +29,18 @@ const Status = () => {
 };
 
 const AccountInformation = ({ account }: { account: TAccount }) => {
+    const {
```

Comment:
@hamid-deriv This  component is not connected to the store and has no access to ui store

Reply:
Fixed.